docs/dev-guides/howto-release-helm-chart.md

```diff
@@ -22,9 +22,10 @@ branch to create `helm/0.3.2` and then `helm/0.3.2_pub` branches.
 After we have a `2.8.0` release of the platform the following files should be updated:
 
 * `deploy/helm/azure-industrial-iot/values.yaml`:
-  * Value of `image:tag` should be updated to so that the chart consumes latest release (`2.8.0`).
+  * Value of `image:tag` should be updated so that the chart consumes latest release (`2.8.0`).
+* `deploy/helm/azure-industrial-iot/Chart.yaml`:
+  * Value of `appVersion` should be changed to align with value of `image:tag` of `values.yaml` (`2.8.0`).
   * Value of `version` should be checked to reflect intended version of next release. For `2.8.0` this should be `0.4.0`.
```

Comment:
Shouldn't we refer to version 2.8.1 (latest release) and 0.4.1 for Helm chart?

Reply:
This is a dev guide on how to release our Helm charts in general. It was written at the time of `2.8.0` release, but it is generic enough that it can be used for future releases as well. And as such, I don't think that we need to update it with each release. Here I am only fixing a bug which was that `appVersion` parameter was listed under `values.yaml` instead of `Chart.yaml`.